apps/meteor/app/lib/server/functions/saveUserIdentity.ts

```diff
@@ -125,7 +126,8 @@ async function updateUsernameReferences({
 		const cursor = Messages.findByMention(previousUsername);
 		for await (const msg of cursor) {
 			const updatedMsg = msg.msg.replace(new RegExp(`@${previousUsername}`, 'ig'), `@${username}`);
-			await Messages.updateUsernameAndMessageOfMentionByIdAndOldUsername(msg._id, previousUsername, username, updatedMsg);
+			const updatedMd = messageTextToAstMarkdown(updatedMsg);
+			await Messages.updateUsernameAndMessageOfMentionByIdAndOldUsername(msg._id, previousUsername, username, updatedMsg, updatedMd);
```

Comment:
Maybe change name of the update function to also include info about `md` field as a better practice 🤔